### DISCUSSION TITLE
N. 2 PC CONNECTION QUESTIONS

### DISCUSSION THREAD

**Discussion by paki59** [apohl79/audiogridder#1234] [General]
Hi from Italy.

I need to know some steps in order to use your beautiful program.
I have a Laptop with Cubase 12 installed and various plug-ins and vst but little ram (16 gb) I bought a 64 gb server computer where I will install various heavy Vst (see Opus Orchestra of East West)....and I would like to connect the two computer.
Questions: can the Lan connection be made with an ethernet cable directly between the two computers without going through a router?...The laptop (master) with Cubase must have AUDIO GRIDDER inserted as a plug in or vst (Midi) to see OPUS ORCHESTRA ON THE SLAVE? IS THE IP NUMBER TO BE INSERTED in the Master slot that of the slave pc? Thank you

**Comment by oldtos** [comment]
With the correct Ethernet cable you can connect them direct....  ..or use a Gigabyte switch or router..
Server goes on computer that you want to run vst from... plugin goes on daw computer...
server will scan for vst... 
on daw computer insert audiogridder as an effect or instrument... use its gui/interface to select vst/vsti

on bottom right of pc screen is audiogridder icon.. right click it .. choose sever to connect to..

**Reply by paki59** [reply]
Good morning. Thanks for your prompt reply. I have done all that you have highlighted, but there is a small problem: the PCs can be seen but I cannot send the midi and/or audio signals to the DAW. Something missing in the set up? Thank you p.s.: but i must install AG server on pc server or only in may principal daw computer?

**Comment by oldtos** [comment]
install server on spare pc ( not daw pc)...   then run audiogridder server ... it will scan vst  common locations ...(you may add other locations to scan in settings)  ... sever will be in system tray (bottom right of deck top screen).... right click on icon... choose scan  if needed
After scan is complete.. server ready...

on DAW pc...... open DAW....  to test.... start new project...  go to effects find E47 ... drop down E47 you will see audiogridder plug in... put audiogridder plugin on test track..

on daw pc go to tray ( tray (bottom right of deck top screen) right click on audiogridder icon ..  it shows server ..select sever
now in daw server name or ip shows in audiogridder plugin.

click ...+ ... choose vst

**Comment by paki59** [comment]
Hi....  what is  E47? I have in insert slot  DAW MASTER ...AG AND AG MIDI AND IN SLAVE PC  ....AG SERVER.

AND ANOTHER PROBLEM  why when i load Opus orchestra in slot AG as Vst  i see  MANY TRACKS SAY  : UNASSIGNED? 
i loaded the samplers of OPus  in Pc server but i don't see anything in the AG slots

**Reply by oldtos** [reply]
install plugin on daw... E47 is category of audiogridder plugin ...like softtube.... eventide ..waves.. E47... <---audio gridder plugins
you use audiogridder plugin ( on daw pc) to access audiogridder server ( second pc)

**Reply by paki59** [reply]
Ok i downloaded  version 1.2.0 for server  and   1.2.0  plugin...are  a lot of days that i am doing test  but  ....Daw do not read  and play  VST OPUS  in server. I see only  OPUS    00  UNASSIGNED  01  UNASSIGNED  02  UNASSIGNED  03 UNASSIGNED ETC ETC....THE VSTS ARE ALSO CONNECTED BUT THERE ARE NO LIBRARIES.....WHY?
EVEN THOUGH THEY ARE UPLOADED ON OPUS

**Comment by oldtos** [comment]
hxxps://www.youtube.com/watch?v=oIaSI-P4O9g

**Reply by paki59** [reply]
hI  I HAVE SEND YOU A PHOTO BY MAIL ...but i think there are problem in yr mail that reject.

In any case Thank you for your patience...It seems that I have found the solution...but first I have to show you what happens connecting (as first solution) the lap top as master and the server pc as slave.....Is all correct but at the end   in the Vst in AG  (Opus Orchestra or Kontakt)  I see all slot  UNASSIGNED.

SECOND SOLUTION:  I REVERSED THE PARTS.

master on desktop and slave on laptop....for now everything works great...i'm doing tests.
TWO QUESTIONS:
1) When it appears 9 stereo 1 mono or 5 stereo 2 mono what should I choose?

2) When I load a vst instrument AUDIO GRIDDER INST...should I open for each VST instance? ie 1 for Kontakt, 1 for Opus, 1 for Spitfire orchestra 1 guitar, etc?...thanks

**Comment by paki59** [comment]
hi  ANY answer?...Thanks.   Also I'm trying to get Play 6 to work with east west libraries but strangely when I load it crashes...maybe because the laptop can't keep the vst image...it shows and disappears it shows and disappears... ( too big?) how can I fix? Is it possible to make the UI image smaller? Thank you

**Comment by oldtos** [comment]
I will install and test  Opus.....

**Reply by damondahl** [reply]
bro. I need help.

**Reply by damondahl** [reply]
https://github.com/apohl79/audiogridder/discussions/1239

**Reply by damondahl** [reply]
here is a link to the discussion
